### PR TITLE
Simplify travis config

### PR DESCRIPTION
**Pull request by hamishforbes**

### CODE REVIEW

**Comment by hamishforbes** [comment]
@pintsized Can you have a look at why Travis isn't running on this anymore?

I've simplified the config so it just uses the official openresty packages which should be a bunch faster too

File: .travis.yml [hunk 1]
```diff
@@ -1,67 +1,21 @@
...
-    - LD_LIBRARY_PATH=$LUAJIT_LIB:$LD_LIBRARY_PATH
-    - TEST_NGINX_SLEEP=0.006
-  matrix:
-    - NGINX_VERSION=1.9.15
-#    - NGINX_VERSION=1.10.0
+  - wget -qO - https://openresty.org/package/pubkey.gpg | sudo apt-key add -
+  - sudo apt-get -y install software-properties-common
+  - sudo add-apt-repository -y "deb http://openresty.org/package/ubuntu $(lsb_release -sc) main"
+  - sudo apt-get update
+  - sudo apt-get install openresty cpanminus
```
Comment (kleisauke) [inline .travis.yml]: Could be more simplified, for e.g.:
```yml
addons:
  apt:
    sources:
      - sourceline: 'deb http://openresty.org/package/ubuntu trusty main'
        key_url: 'https://openresty.org/package/pubkey.gpg'
    packages:
      - openresty
      - libtest-nginx-perl
      - cpanminus
```

**Comment by pintsized** [comment]
Travis failures are the outbound requests to `google.com` in `t/14-host-header.t`

**Comment by pintsized** [comment]
Closing as no longer relevant with the docker test runner

**Pull request closed without merging**